cli/stork-publisher-agent/runner.go

```diff
@@ -15,7 +15,9 @@ type PublisherAgentRunner[T Signature] struct {
 	logger                  zerolog.Logger
 	priceUpdateCh           chan PriceUpdate
 	signedPriceBatchCh      chan SignedPriceUpdateBatch[T]
-	outgoingConnections     map[ConnectionId]*OutgoingWebsocketConnection[T]
+	registryClient          *RegistryClient
+	brokerMap               map[BrokerPublishUrl]map[AssetId]struct{}
```

Comment:
this stores the previously configured state of the world so we can see what needs to change when we get the latest configuration from the registry